server/web/templates/auth/email/confirm.txt

```diff
@@ -0,0 +1,6 @@
+Dear {{ user.first_name }},
```

Comment:
- Should be `confirm.txt.j2`
- need `confirm.html.j2`  - this does not inherit from base btw
- templates for email should be under `templates/email`